docs/enterprise-setup/multi-region.md

```diff
@@ -0,0 +1,455 @@
+  AWS_SECRET_MANAGER_ACCESS_KEY_ID: your-aws-secret-manager-access-key
+  AWS_SECRET_MANAGER_SECRET_ACCESS_KEY: your-aws-secret-manager-secret-key
+  S3_ACCESS_KEY_ID: your-s3-access-key
+  S3_SECRET_ACCESS_KEY: your-s3-secret-key
+
+  # GCP Secrets
+  gcp.json: <CREDENTIALS_JSON_BLOB>
+```
+
+Apply your secrets manifest in your terminal with `kubectl`: `kubectl apply -f <file>.yaml -n <namespace>`.
+
+You can also use `kubectl` to create the secret directly from the command-line tool:
+
+```bash
+kubectl create secret generic airbyte-config-secrets \
+  --from-literal=license-key='' \
+  --from-literal=data_plane_client_id='' \
+  --from-literal=data_plane_client_secret='' \
+  --from-literal=s3-access-key-id='' \
+  --from-literal=s3-secret-access-key='' \
+  --from-literal=aws-secret-manager-access-key-id='' \
+  --from-literal=aws-secret-manager-secret-access-key='' \
+  --from-file=gcp.json
+  --namespace airbyte
+```
+
+</TabItem>
+</Tabs>
+</details>
+
+## 5. Create your deployment values {#step-5}
+
+Add the following overrides to a new `values.yaml` file.
+
+```yaml title="values.yaml"
+airbyteUrl: https://airbyte.example.com # Base URL for the control plane so Airbyte knows where to authenticate
+
+edition: enterprise # Required for Self-Managed Enterprise
+
+# Logging:
+#  level: DEBUG
+
+dataPlane:
+  # Used to render the data plane creds secret into the Helm chart.
+  secretName: airbyte-config-secrets
+  id: "preview-data-plane"
+
+  # Describe secret name and key where each of the client ID and secret are stored
+  clientIdSecretName: airbyte-config-secrets
+  clientIdSecretKey: "DATA_PLANE_CLIENT_ID"
+  clientSecretSecretName: airbyte-config-secrets
+  clientSecretSecretKey: "DATA_PLANE_CLIENT_SECRET"
+
+# Describe the secret name and key where the Airbyte license key is found
+enterprise:
+  secretName: airbyte-config-secrets
+  licenseKeySecretKey: AIRBYTE_LICENSE_KEY
+
+# S3 bucket secrets/config
+# Only set this section if the control plane has also set these values.
+storage:
+  secretName: airbyte-config-secrets
+  type: "s3"
+  bucket:
+    log: my-bucket-name
+    state: my-bucket-name
+    workloadOutput: my-bucket-name 
+  s3:
+    region: "us-west-2"
+    authenticationType: credentials
+    accessKeyIdSecretKey: S3_ACCESS_KEY_ID
+    secretAccessKeySecretKey: S3_SECRET_ACCESS_KEY
+
+# Secret manager secrets/config
+# Only set this section if the control plane has also set these values.
+secretsManager:
+  secretName: airbyte-config-secrets
+  type: AWS_SECRET_MANAGER
+  awsSecretManager:
+    region: us-west-2 
+    authenticationType: credentials
+    accessKeyIdSecretKey: AWS_SECRET_MANAGER_ACCESS_KEY_ID 
+    secretAccessKeySecretKey: AWS_SECRET_MANAGER_SECRET_ACCESS_KEY
+```
+
+## 6. Deploy your data plane {#step-6}
+
+In your terminal:
+
+```bash
+helm upgrade --install airbyte-enterprise airbyte/airbyte-data-plane --version 1.6.0 -n airbyte-dataplane --values values.yaml
```

Comment:
FYI: this assumes the name of the namespace is `airbyte-dataplane` which may not be true as we did not ask the end user to create a new namespace